README.md

```diff
@@ -328,6 +328,35 @@ to your chosen fungible token for now. Then, use that public path for your royal
 and in the future, you will be able to easily update the link at that path to use the
 fungible token switchboard instead.
 
+## Contract metadata
+
+Now that contract borrowing is released, you can also implement the [Resolver](./contracts/Resolver.cdc) interface on your contract
+and resolve views from there. As an example, you might want to allow your contract to resolve NFTCollectionData and NFTCollectionDisplay
+so that platforms do not need to find an NFT that belongs to your contract to get information about how to set up or show your collection.
+
+```cadence
+import ViewResolver from 0xf8d6e0586b0a20c7
+import MetadataViews from 0xf8d6e0586b0a20c7
+
+pub fun main(addr: Address, name: String): AnyStruct? {
```

Comment:
Should this return `StoragePath?` instead of `AnyStruct?`? It would also be helpful to add this example to the `scripts/`directory.

Reply:
haha yeah this is `AnyStruct?` because i was playing around with what to make the test into, I get annoyed at changing the actual return format so I just leave it like this sometimes

Happy to change it, the main showcase is just how to resolve views in this way, what it returns isn't as big of a deal.

> It would also be helpful to add this example to the scripts/directory.

Noted!